packages/graphql/src/utils/global-ids.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { GraphQLID } from "graphql";
+import { base64, unbase64 } from "graphql-relay/utils/base64";
+
+export function toGlobalId(typeName: string, field: string, id: string | number): string {
```

Comment:
**suggestion (non-blocking):** wondering whether this is a touch nicer with named arguments in an object?

```suggestion
export function toGlobalId({ typename, field, id }: { typename: string, field: string, id: string | number }): string {
```

Reply:
Up to you!

Reply:
Let's do it!! That way you can jump between `fromGlobalId` and `toGlobalId` without spreading the resulting object.